### PR TITLE
Introducing github actions to ocs-ci

### PR DESCRIPTION
**Pull request by vavuthu**
Fixes: #3312 

Signed-off-by: vavuthu <vavuthu@redhat.com>

### CODE REVIEW

**Comment by clacroix12** [comment]
@vavuthu Nice! Do we have any executions yet? If so where can I view the log?

File: .github/workflows/pr_checks.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: "PR Checks"
+# Triggers the workflow on pull request events
+on: pull_request
+
+jobs:
+  pr_checks:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ 3.6, 3.7 ]
```
Comment (AviadP) [inline .github/workflows/pr_checks.yml:10]: do we still need python 3.6?

Comment (petr-balogh) [reply .github/workflows/pr_checks.yml:10]: We didn't move our jenkins agents yet to use higher python. So we will remove support for 3.6 once we will finish with that.  Discussed on last automation mtg.

**Comment by vavuthu** [comment]
> @vavuthu Nice! Do we have any executions yet? If so where can I view the log?

I have tested for push event which is triggered on my fork repo.

here is the logs for the actions:

https://github.com/vavuthu/ocs-ci/runs/1413644224?check_suite_focus=true

```
tox: collectonly
___________________________________ summary ____________________________________
  black: commands succeeded
  py37: commands succeeded
  flake8: commands succeeded
  docs: commands succeeded
  collectonly: commands succeeded
  congratulations :)
```

**Review by petr-balogh — APPROVED** [review]
Perfect!

**Review by dahorak — APPROVED** [review]

**Pull request merged**